src/spesh/optimize.c

```diff
@@ -2279,6 +2279,7 @@ static void optimize_bb_switch(MVMThreadContext *tc, MVMSpeshGraph *g, MVMSpeshB
         case MVM_OP_dispatch_n:
         case MVM_OP_dispatch_s:
         case MVM_OP_dispatch_i:
+        case MVM_OP_dispatch_u:
             MVM_spesh_disp_optimize(tc, g, bb, p, ins, &next_ins);
             break;
         case MVM_OP_sp_guard:
```

Comment:
Shouldn't there also be a case for `MVM_OP_sp_runbytecode_u` added a couple lines below?

Reply:
Oh, yes, indeed! Nice catch!